install/install.py

```diff
@@ -102,7 +130,8 @@ def install(self, cmd_arguments):
             distro_num = 2
         elif "debian" in distro.lower():
             distro_num = 3
-
+        elif self.is_compatible():
```

Comment:
Why an extra `elif` statement? A clean solution would be to replace the debian thing (above it) with this.